File: app/src/main/java/com/purpletealabs/bitcoinapp/activities/BaseActivity.java
package com.purpletealabs.bitcoinapp.activities;

import android.support.design.widget.Snackbar;
import android.support.v7.app.AppCompatActivity;

public class BaseActivity extends AppCompatActivity {

    public void showSnackBar(String message) {
        Snackbar.make(findViewById(android.R.id.content), message, Snackbar.LENGTH_LONG).show();
    }
}

File: app/src/main/java/com/purpletealabs/bitcoinapp/activities/BaseActivity.kt
package com.purpletealabs.bitcoinapp.activities

import android.support.design.widget.Snackbar
import android.support.v7.app.AppCompatActivity
import android.view.View

open class BaseActivity : AppCompatActivity() {

    fun showSnackBar(message: String) {
        Snackbar.make(findViewById<View>(android.R.id.content), message, Snackbar.LENGTH_LONG).show()
    }
}
